.vscode/react-components-for-you.code-workspace

```diff
@@ -0,0 +1,23 @@
+{
+	"folders": [
+		{
+			"path": ".."
+		}
```

Comment:
Does "path": ".." mean that we're only applying this to folders and files two levels down from root?